gap/IdFuncP4Q.gi

```diff
@@ -796,9 +796,6 @@ SOTRec.IdGroupP4Q := function(group, n, fac)
         elif (p + 1) mod q = 0 and q > 2 then
```

Comment:
Note that for 48, we have p=2, q=3, so the above check triggers, and hence the separate n=48 case below is never reached